include/seqan3/range/container/small_string.hpp

```diff
@@ -309,6 +337,65 @@ class small_string : public small_vector<char, capacity_ + 1>
         return str();
     }
     //!\}
+
+    /*!\name Input/output
+     * \{
+     */
+
+    /*!\brief Formatted output for the seqan3::small_string.
+     * \param[in,out] os  The std::basic_ostream to write to.
+     * \param[in]     str The seqan3::small_string to read from.
+     * \returns `os`.
+     *
+     * \details
+     *
+     * Internally calls `os << str.str()`.
+     */
+    friend std::ostream & operator<<(std::ostream & os, small_string const & str)
+    {
+        os << str.str();
+        return os;
+    }
+
+    /*!\brief Formatted input for the seqan3::small_string.
+     * \param[in,out] is  The std::basic_istream to read from.
+     * \param[out]    str The seqan3::small_string to write to.
+     * \returns `is`.
+     *
+     * \details
+     *
+     * Reads at most seqan3::small_string::small_size characters from the stream.
```

Comment:
```suggestion
     * Reads at most seqan3::small_string::max_size characters from the stream.
```

Reply:
What? There is no `small_size` member function 😱